spec/models/project_spec.rb

```diff
@@ -24,4 +24,37 @@
       end
     end
   end
+
+  describe "Project.export" do
+    it "exports project data" do
+      project_params = {
+        about: "Some text about the project.",
+        footer: "Copyright information and such.",
+        help: "Help text for the project.",
+        logo_ap: "https://static.concord.org/projects/logos/ap/project-1.png",
+        logo_lara: "https://static.concord.org/projects/logos/lara/project-1.png",
+        project_key: "test-project-1",
+        title: "Test Project 1",
+        url: "https://concord.org/projects/test-project-1/"
+      }
+      project = Project.create(project_params)
+      expect(project.export.to_json).to eq(project_params.to_json)
+    end
+  end
+
+  describe "Project.find_or_create" do
+    project_data = {project_key: "default-project", title: "Default Project"}
+    it "finds a project when it exists" do
```

Comment:
Based on the test coverage report. It seems like this test isn't actually finding a project, but it is instead creating a new one.

Reply:
looks like this works now